From 2e1c925693871dfba76845b82bb653dc09b2de97 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Mon, 24 Feb 2014 08:02:28 -0500
Subject: [PATCH] units/serial-getty@.service: add [Install] section

This makes it easier to manually enable and disable
specific gettys, and also mirrors getty@.service.

http://lists.freedesktop.org/archives/systemd-devel/2014-February/017329.html

Resolves: #1118230
---
 units/serial-getty@.service.m4 | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/units/serial-getty@.service.m4 b/units/serial-getty@.service.m4
index e32c6b7..aea078f 100644
--- a/units/serial-getty@.service.m4
+++ b/units/serial-getty@.service.m4
@@ -33,3 +33,6 @@ TTYVHangup=yes
 KillMode=process
 IgnoreSIGPIPE=no
 SendSIGHUP=yes
+
+[Install]
+WantedBy=getty.target
